search/tests/mock_search_engine.py

```diff
@@ -47,6 +48,11 @@ def _find_field(doc, field_name):
         return field_value
 
 
+def _is_iterable(item):
```

Comment:
@e-kolpakov  There's a warning in the docs you've added about searching multivalue fields. This code here seems oriented toward doing something with that, though perhaps I misunderstand. Is the warning accurate?